ingestion_tools/scripts/common/config.py

```diff
@@ -266,6 +266,8 @@ def resolve_output_path(self, key: str, obj: BaseImporter) -> str:
             "deposition_metadata": "depositions_metadata/{deposition_name}/deposition_metadata.json",
             "deposition_keyphoto": "depositions_metadata/{deposition_name}/Images",
         }
+        paths["collection_metadata"] = paths["rawtilt"]
```

Comment:
Could we hardcode this in the map for consistency?

Reply:
I think the paths variable should live closer the classes that use them. Updating this file when adding a new ingest config entity is not intuitive at all.